CONTRIBUTING.md

```diff
@@ -44,6 +44,9 @@ we operate with a limited contribution model.
 
 ## Becoming a contributor
 
+Review the guide for
+[Onboarding to Librarian](https://github.com/googleapis/librarian/blob/main/doc/onboarding.md).
```

Comment:
I *suspect* this can just be `[Onboarding to Librarian](doc/onboarding.md)` but I'm fine to leave it as is.